### PR TITLE
Add my card info

### PR DESCRIPTION
**Pull request by zelalemMK**

### CODE REVIEW

**Comment by zelalemMK** [comment]
This is cool.

File: index.html [hunk 1]
```diff
@@ -108,6 +108,39 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="https://i.am.ai/" target="_blank" title="Third resource">
+                  Resource 3
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+
```
Comment (TenTraicion) [inline index.html:141]: add ` after the end of your card

File: index.html [hunk 2]
```diff
@@ -108,6 +108,39 @@ <h2>A project for first-time contributions</h2>
         <!-- ============================================================== -->
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
 
+
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (TenTraicion) [inline index.html:112]: edit `<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________ -->` at the start of your card
change `TEMPLATE: MAKE A COPY` with your name

**Review by TenTraicion — CHANGES_REQUESTED** [review]
Kindly resolve the issues!

**Pull request merged**